README.md

```diff
@@ -151,6 +151,7 @@ Please have a look to the section [on building the source code](doc/manuals/admi
 
 * [Advanced Programming](doc/manuals/user/README.md)
 * [Installation and administration](doc/manuals/admin/README.md)
+  * [HA Architecture and Deployment](doc/manuals/admin/extra/ha.md)
```

Comment:
Instead of addint the link here, I'd suggest to adding it in doc/manuals/admin/README.md, as it is actually part of the Installation & Administration topics.

It is an advanced topic, so I'd suggest to add it just after "Performance Tunning" line.

Morevoer, I think it should be a good idea to have this documentation also in the RTD documentation. The mkdocs.yml document should be modified accordingly.